### PR TITLE
Flask skeleton webpage

### LINKED ISSUE #13: Add template for web app
Add the barebones HTML template to be used for the main page of the app, ideally under `web/templates`. Render the template when the user goes to `/`.

### PR DESCRIPTION
**Pull request by chrisnguyn**
Barebones webpage so we can start with map rendering, etc.

There's a lot of stuff in that .css file we currently don't use but in case we do need it in the future I'll leave it in there for now

### CODE REVIEW

**Comment by chrisnguyn** [comment]
dammit I forgot about the linter

if you guys approve, lmk and I'll just add those two newlines afterwards

File: web/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+        <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+        <link rel="shortcut icon" href="icons/favicon.ico" type="image/x-icon">
```
Comment (YKo20010) [inline web/templates/index.html]: move above scripts

File: web/static/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  height: 100%;
+  width: 100%;
+} */
+
+.nav-item {
+    border-bottom: 1.5px solid #000;
+    border-radius: 0 !important;
+  }
+  
+  .nav-item:nth-child(1) {
```
Comment (YKo20010) [inline web/static/styles/style.css]: spacing inconsistent for the headers

File: web/main.py [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+# def index2():
+#     return 'Dicking around with Flask'
+
+# @app.route('/<name>')
+# def index3(name):
+#     return 'Hello {}'.format(name)
+
+# @app.route('/anything')
+# def index4():
+#     return redirect(url_for('index3', name='Kai'))
```
Comment (YKo20010) [inline web/main.py]: Remove unnecessary commented code or at least make the strings more PG

File: web/static/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    margin-left: auto !important;
+    margin-right: auto !important;
+    max-width: 80% !important;
+    /* width: 50% !important; */
+    /* max-width: 75% !important; */
+  }
+  
+  /* LYNN */
+  
+  .borderr {
```
Comment (YKo20010) [inline web/static/styles/style.css]: border instead of borderr if this can be changed

File: web/static/styles/style.css [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+      }
+      50% {
+          stroke: #fff;
+      }
+      100% {
+          stroke: #222;
+      }
+  }
+  
+  .coverr {
```
Comment (YKo20010) [inline web/static/styles/style.css]: cover instead of coverr if this can be changed

File: web/main.py (hunk 3 above)
Comment (kx-chen) [reply web/main.py]: is my name not pg 😡

Comment (chrisnguyn) [reply web/main.py]: LMAOOOOOOOO

**Review by YKo20010 — APPROVED** [review]

**Review by kx-chen — APPROVED** [review]

**Pull request merged**